server/service.go

```diff
@@ -0,0 +1,11 @@
+// +build !windows
+// Copyright 2012-2017 Apcera Inc. All rights reserved.
+
+package server
+
+// Run starts the NATS server. This wrapper function allows Windows to add a
+// hook for running NATS as a service.
+func Run(server *Server) error {
+	server.Start()
```

Comment:
That sounds good here.  Services writing to the syslog is standard behavior; if someone wants something otherwise, we can address it then.